<ui version="4.0" >
 <class>config</class>
 <widget class="QWidget" name="config" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize" >
   <size>
    <width>0</width>
    <height>480</height>
   </size>
  </property>
  <property name="styleSheet" >
   <string notr="true" />
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QGroupBox" name="groupBox_2" >
     <property name="title" >
      <string>Image</string>
     </property>
     <property name="flat" >
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout" >
      <item>
       <widget class="QComboBox" name="pictureComboBox" >
        <item>
         <property name="text" >
          <string>Image</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Slideshow</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Picture of the day</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QStackedWidget" name="stackedWidget" >
        <property name="minimumSize" >
         <size>
          <width>350</width>
          <height>51</height>
         </size>
        </property>
        <property name="currentIndex" >
         <number>0</number>
        </property>
        <widget class="QWidget" name="page" >
         <property name="geometry" >
          <rect>
           <x>0</x>
           <y>0</y>
           <width>454</width>
           <height>230</height>
          </rect>
         </property>
         <layout class="QGridLayout" >
          <item row="0" column="0" >
           <layout class="QHBoxLayout" >
            <item>
             <widget class="QLabel" name="picLabel" >
              <property name="text" >
               <string>Picture:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="KUrlComboRequester" name="picRequester" >
              <property name="sizePolicy" >
               <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="acceptDrops" >
               <bool>true</bool>
              </property>
              <property name="filter" >
               <string>*.png *.jpeg *.jpg *.svg *.svgz</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="0" >
           <layout class="QVBoxLayout" >
            <item>
             <spacer>
              <property name="orientation" >
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0" >
               <size>
                <width>421</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <layout class="QHBoxLayout" >
              <item>
               <spacer>
                <property name="orientation" >
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeType" >
                 <enum>QSizePolicy::MinimumExpanding</enum>
                </property>
                <property name="sizeHint" stdset="0" >
                 <size>
                  <width>40</width>
                  <height>91</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QLabel" name="monitorLabel" >
                <property name="text" >
                 <string>TextLabel</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer>
                <property name="orientation" >
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeType" >
                 <enum>QSizePolicy::MinimumExpanding</enum>
                </property>
                <property name="sizeHint" stdset="0" >
                 <size>
                  <width>40</width>
                  <height>101</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item>
             <spacer>
              <property name="orientation" >
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0" >
               <size>
                <width>438</width>
                <height>16</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="page_3" >
         <property name="geometry" >
          <rect>
           <x>0</x>
           <y>0</y>
           <width>454</width>
           <height>230</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout" >
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout" >
            <item>
             <layout class="QHBoxLayout" >
              <item>
               <widget class="KListWidget" name="slideShowDirList" />
              </item>
              <item>
               <layout class="QVBoxLayout" >
                <item>
                 <widget class="KPushButton" name="addDirButton" >
                  <property name="text" >
                   <string>&amp;Add Folder...</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="KPushButton" name="removeDirButton" >
                  <property name="enabled" >
                   <bool>false</bool>
                  </property>
                  <property name="text" >
                   <string>&amp;Remove Folder</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer>
                  <property name="orientation" >
                   <enum>Qt::Vertical</enum>
                  </property>
                  <property name="sizeHint" stdset="0" >
                   <size>
                    <width>20</width>
                    <height>40</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2" >
            <item>
             <widget class="QCheckBox" name="recursiveCheckBox" >
              <property name="text" >
               <string>&amp;Include Subfolders</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="randomCheckBox" >
              <property name="text" >
               <string>Randomize</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" >
            <item>
             <widget class="QLabel" name="label_2" >
              <property name="text" >
               <string>Change images every:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTimeEdit" name="slideShowDelay" >
              <property name="dateTime" >
               <datetime>
                <hour>0</hour>
                <minute>0</minute>
                <second>0</second>
                <year>2000</year>
                <month>1</month>
                <day>1</day>
               </datetime>
              </property>
              <property name="currentSection" >
               <enum>QDateTimeEdit::HourSection</enum>
              </property>
              <property name="displayFormat" >
               <string>hh 'Hours' mm 'Mins' ss 'Secs'</string>
              </property>
              <property name="time" >
               <time>
                <hour>0</hour>
                <minute>0</minute>
                <second>0</second>
               </time>
              </property>
             </widget>
            </item>
            <item>
             <spacer>
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0" >
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="page_2" >
         <property name="geometry" >
          <rect>
           <x>0</x>
           <y>0</y>
           <width>454</width>
           <height>230</height>
          </rect>
         </property>
         <widget class="QComboBox" name="potdComboBox" >
          <property name="geometry" >
           <rect>
            <x>10</x>
            <y>30</y>
            <width>271</width>
            <height>27</height>
           </rect>
          </property>
         </widget>
         <widget class="QLabel" name="selectPoTDlabel" >
          <property name="geometry" >
           <rect>
            <x>10</x>
            <y>10</y>
            <width>158</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text" >
           <string>Select Picture of the day source:</string>
          </property>
         </widget>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Appearance</string>
     </property>
     <property name="flat" >
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout" >
      <item>
       <layout class="QVBoxLayout" >
        <item>
         <widget class="QCheckBox" name="roundCheckBox" >
          <property name="text" >
           <string>Use rounded corners</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="shadowCheckBox" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text" >
           <string>Show shadow</string>
          </property>
          <property name="checked" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="frameCheckBox" >
          <property name="text" >
           <string>Show frame</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" >
          <item>
           <widget class="QLabel" name="changeFrameColorLabel" >
            <property name="enabled" >
             <bool>false</bool>
            </property>
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Preferred" hsizetype="Minimum" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text" >
             <string>Frame color:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="KColorButton" name="changeFrameColor" >
            <property name="enabled" >
             <bool>false</bool>
            </property>
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Fixed" hsizetype="MinimumExpanding" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="color" >
             <color>
              <red>70</red>
              <green>90</green>
              <blue>130</blue>
             </color>
            </property>
            <property name="defaultColor" >
             <color>
              <red>70</red>
              <green>90</green>
              <blue>130</blue>
             </color>
            </property>
           </widget>
          </item>
          <item>
           <spacer>
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0" >
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KColorButton</class>
   <extends>QPushButton</extends>
   <header>kcolorbutton.h</header>
  </customwidget>
  <customwidget>
   <class>KListWidget</class>
   <extends>QListWidget</extends>
   <header>klistwidget.h</header>
  </customwidget>
  <customwidget>
   <class>KPushButton</class>
   <extends>QPushButton</extends>
   <header>kpushbutton.h</header>
  </customwidget>
  <customwidget>
   <class>KUrlComboRequester</class>
   <extends>KUrlRequester</extends>
   <header>kurlrequester.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>frameCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>changeFrameColor</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>469</x>
     <y>33</y>
    </hint>
    <hint type="destinationlabel" >
     <x>166</x>
     <y>94</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pictureComboBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>stackedWidget</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>184</x>
     <y>220</y>
    </hint>
    <hint type="destinationlabel" >
     <x>184</x>
     <y>264</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>frameCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>changeFrameColorLabel</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>239</x>
     <y>418</y>
    </hint>
    <hint type="destinationlabel" >
     <x>54</x>
     <y>447</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
